src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -91,6 +94,23 @@ export class SelectBox extends vsSelectBox {
 		this._register(focusTracker);
 		this._register(focusTracker.onDidBlur(() => this._hideMessage()));
 		this._register(focusTracker.onDidFocus(() => this._showMessage()));
+		// Stop propagation - we've handled the event already and letting it bubble up causes issues with parent
+		// controls handling it (such as dialog pages)
+		this.onkeydown(this.selectElement, (e: IKeyboardEvent) => {
```

Comment:
This should be fine but to be sure, can you please validate once that the dialogs/wizards that are working now don't start behaving weird. These two should be able to cover this,
1. Restore wizard directly using select box 
2. DacFx wizard (ex. extract flow) using it through .component

Please let me know if you need help validating those.

Reply:
Verified that things seem to be working fine there.